addons/project/models/project_task.py

```diff
@@ -1890,7 +1890,7 @@ def action_convert_to_subtask(self):
                 'type': 'ir.actions.act_window',
                 'res_model': 'project.task',
                 'res_id': self.id,
-                'views': [(self.env.ref('project.project_task_convert_to_subtask_view_form', False).id, 'form')],
+                'views': [(self.env.ref('project.project_task_convert_to_subtask_view_form').id, 'form')],
```

Comment:
```suggestion
                'views': [(self.env.ref('project.project_task_convert_to_subtask_view_form', raise_if_not_found=False).id, 'form')],
```

Reply:
Sorry, but since `ref` returns `None` when not finding a record (if we don't have an `ir.model.data` record we don't know the `res_model` so it can't return an empty recordset) so the `None.id` call raises an ugly `AttributeError` instead of a much better `ValueError` with "Record not found".
Same goes for other comments.